mmv1/products/artifactregistry/api.yaml

```diff
@@ -131,3 +131,26 @@ objects:
       name: updateTime
       description: The time when the repository was last updated.
       output: true
+    - !ruby/object:Api::Type::NestedObject
+      name: 'maven_config'
+      description: |-
+        Maven Config defines maven related repository details.
+        Provides additional configuration details for repositories of the maven
+        format type.
+      properties:
+        # Maven properties.
+        - !ruby/object:Api::Type::Boolean
+          name: 'allow_snapshot_overwrites'
+          description: |-
+            The repository with this flag will allow publishing the same
+            snapshot versions.
+        - !ruby/object:Api::Type::Enum
+          name: 'version_policy'
+          description: |-
+            Version policy defines the versions that the registry will accept.
+          input: true
```

Comment:
The versionPolicy is here: https://cloud.google.com/artifact-registry/docs/reference/rest/v1/projects.locations.repositories#MavenRepositoryConfig

Reply:
@Gridasov to clarify: Marking this as an `input` field means that changing it will cause the entire resource to be deleted and recreated, because the field can only be set at create time. I don't see anywhere in the docs that states that this field can't be updated.

Reply:
If this field (or allowSnapshotOverwrites) _can_ be updated, it might be worth adding that to an update test for this resource, since API patch operations sometimes need some tweaking before they work correctly.

Reply:
Both should not be modified. Changed allowSnapshotOverwrites to have "input: true".